applications/Unity.GrantManager/modules/Unity.Payments/src/Unity.Payments.Application.Contracts/Integrations/Cas/Invoice.cs

```diff
@@ -0,0 +1,109 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+
+namespace Unity.Payments.Integrations.Cas
+{
+    public class Invoice
+    {
+        [Required]
+        [MaxLength(25)]
+        public string? invoiceType { get; set; } = "Standard";

```

Comment:
lets update the Casing of these properties and use the json property name to serialize as needed pls

Reply:
Done, CAS expects these as lower case and throws an exception for the cap case. Had to add decorator property to resolve this and move away from NewtonSoft JSON serialization. The code base I coppied this from must have had the same issue and made them lower case for conveniance of conversion.